### DISCUSSION TITLE
UM780XTX - Ryzen 7 7840HS

### DISCUSSION THREAD

**Discussion by Hitcher99** [isc30/ryzen-gpu-passthrough-proxmox#15] [General]
Hi all, i also tested the GPU Passtrough with the Minisforum UM780XTX
Bios/UEFI Version: 1.0.4 (latest for the UM780XTX)
CPU: AMD Ryzen 7 7840HS
GPU: AMD Radeon 780M

Proxmox VE 8.1 --> NO CHANCE -- PvE Host crash at VM Start!
Proxmox VE 8.0.4  --> WORKS FINE!! (Kernel: Linux 6.2.16-3-pve)
VM: Windows 11 Pro with OVMF/UEFI
So there musst be something wrong/changed with Proxmox VE 8.1!
The whole GPU Passtrough thing seems on all Platforms (Unraid/ProxmoxVE/Hyper-V/VMware) i have tested buggy as H3ll! 
In my opionion its not really worth it! Too many trouble!
It only works when the PvE Host is fresh started and the Windows VM Boots as first!
If you Shutdown the VM and Start it again --> Error43! (Reset Patch/Service seems not to work)
Its buggy as H3ll and i think it will never get stable at all.

![Screenshot 2024-01-07 220604](https://github.com/isc30/ryzen-7000-series-proxmox/assets/28291540/ca032181-d278-4bbd-8d61-3384d569659b)

Here is my fresh vbios dump from the Ryzen 7840HS APU: 
[vbios_1002_15bf.zip](https://github.com/isc30/ryzen-7000-series-proxmox/files/13854932/vbios_1002_15bf.zip)

Best Regards
Hitcher99

**Comment by dsantos21** [comment]
Not sure if you have ever solved this error 43. I've bought and setup an um780xtx myself and Win11 VM. Radeon reset was not working, but then realized that it doesn't account for the device "AMD Audio CoProcessor".

When I disabled it manually prior to boot, Radeon Reset works. The fix would be to update Radeon Reset to include this new device. I'll try it in a few days!

Not sure if this is something already well-known, sorry if just repeating something obvious, and for my english... also, typing on mobile doesn't help either :)

**Comment by mntyfrsh** [comment]
UM780 XTX works fine with GPU passthrough. Details here:

https://github.com/isc30/ryzen-7000-series-proxmox/issues/16

**Comment by josedrribera** [comment]
Por fin me funciono!!!! En Proxmox 9.0.11!! En un minipc Gmkteck K6 con R7 7840HS!! Con el vbios_1002_15bf.zip de @Autoestopista99 que está en el primer post!! Con Kernel: 6.2.16-5-pve
Este es mi archivo 102.conf, tenéis que modificar los ID's vuestros.

> agent: 1
balloon: 0
bios: ovmf
boot: order=hostpci0
cores: 8
cpu: host,hidden=1,flags=+virt-ssbd;+amd-ssbd;+amd-no-ssb;+aes
efidisk0: local-lvm:vm-102-disk-0,efitype=4m,pre-enrolled-keys=1,size=4M
hookscript: local:snippets/amd-reset-780m.pl
hostpci0: 0000:65:00.0,pcie=1,romfile=vbios_1002_15bf.bin,x-vga=1
hostpci1: 0000:65:00.1,pcie=1,romfile=AMDGopDriver_7840hs.rom
ide0: local:iso/virtio-win-0.1.285.iso,media=cdrom,size=771138K
ide2: local:iso/Win11_25H2_Spanish_x64.iso,media=cdrom,size=7564268K
machine: pc-q35-8.1
memory: 8192
meta: creation-qemu=10.0.2,ctime=1761926625
name: Windows11
net0: virtio=BC:24:11:B0:10:47,bridge=vmbr0,firewall=1,queues=8
numa: 0
ostype: win11
scsi0: local-lvm:vm-102-disk-2,iothread=1,size=256G,ssd=1
scsihw: virtio-scsi-single
smbios1: uuid=EL ID POR DEFECTO
sockets: 1
tpmstate0: local-lvm:vm-102-disk-1,size=4M,version=v2.0
vga: std
vmgenid: EL ID POR DEFECTO

He tenido que instalar también vendor-reset, radeonresetbugfix y no recuerdo si algo más... Llevo una semana para hacerlo funcionar.

Espero que os sirva.

Gracias...
